### DISCUSSION TITLE
Theme specified in ui-lovelace.yaml not being applied

### DISCUSSION THREAD

**Discussion by m90att** [UI-Lovelace-Minimalist/UI#1135] [Q&A]
Hi,

I'm just getting started with this and I've found that the theme specified in the yaml file is not being applied to the minimalist views.

I followed @EverythingSmartHome's youtube guide for advanced install and have left the **theme** line as I want to use the minimalist theme only on the minimalist views/pages.

```
title: "Home"
theme: "minimalist-desktop"
background: "var(--background-image)"
views:
  - !include views/home.yaml
```

However this is not being applied.  The only way to do it is to select minimalist-desktop as my overall theme, which obviously knocks out the non-minimalist views/sections that I have.

Can anyone see where I might be going wrong?

Thanks, Matt

**Comment by ingeniumdesign** [comment]
here is my config:

**ui_lovelace_minimalist/dashboard/ui-lovelace.yaml**

```
---
button_card_templates: !include_dir_merge_named "../../custom_components/ui_lovelace_minimalist/__ui_minimalist__/ulm_templates/"

title: "Dashboard Default"
theme: "minimalist-desktop"
background: "var(--background-image)"
views:
  - !include "default/dashboard.yaml"
  - !include "default/mainhallway.yaml"
  - !include "default/livingroom.yaml"

```

**ui_lovelace_minimalist/dashboard/default/dashboard.yaml**

```
title: "Dashboard"
path: "dashboard"
icon: "mdi:home-assistant"
id: "dashboard"
theme: "minimalist-desktop"
cards:
```

i think, you cant edit the foldername, i think the folder name must be ui_lovelace_minimalist!

**Reply by m90att** [reply]
Hi, my ui-lovelace.yaml file is in ui_lovelace_minimalist\dashboard\ui-lovelace.yaml too, so I'm not sure it's a file location issue.  I'm wondering why specifying `theme: "minimalist-desktop"` in ui-lovelace.yaml is not working

**Reply by PieterBoeren** [reply]
For anyone who is looking for this: it works more or less (it doesn't hide the top bar) if you apply the theme per view:

```yaml
views:
  - title: "Example View"
    theme: "minimalist-desktop"
```
